### PR TITLE
The hyphen text postcard

### PR DESCRIPTION
**Pull request by jotaen**

### CODE REVIEW

File: content/posts/2020-07-14-ancient-http/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+image = "/posts/2020-07-14-ancient-http/shakespeare.jpg"
+image_info = "The only preserved portrait painting of Lord Timothy Learnersbee (who by mistake gets often confused with William Shakespeare)"
+image_colouring = "10"
+image_offset = "20%"
+id = "mOGE3"
+url = "mOGE3/lord-timothy-and-the-hyphen-text-postcard"
+aliases = ["mOGE3"]
++++
+
+The origin of the World Wide Web gets commonly associated with the scientific activities at CERN in the 1980s and ’90s. What few people do know, however, is that there briefly existed a transregional network a few centuries ago somewhere in Britain, which had remarkable similarities with the web as we know it today. Unfortunately, almost no testimonies got passed on about this piece of history, so over time it mostly fell into oblivion. I only know about this legend from hearsay as well, so I tried my best to recall it and to put it into writing for posterity.
```
Comment (adrigispen) [inline content/posts/2020-07-14-ancient-http/index.md:15]: I'd say 'is', not gets :)

Comment (adrigispen) [reply content/posts/2020-07-14-ancient-http/index.md:15]: The origin of the World Wide Web is commonly associated

Comment (adrigispen) [reply content/posts/2020-07-14-ancient-http/index.md:15]: which had remarkable similarities to the web

Comment (adrigispen) [reply content/posts/2020-07-14-ancient-http/index.md:15]: almost no testimonies were passed on

File: content/posts/2020-07-14-ancient-http/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Grafting in the library
+
+The story[^1] takes place in the mid 17th century in southern Great Britain, more precisely in a county governed by a wealthy family that went by the name Learnersbee. Son and heir was Lord Timothy, an aristocratic but good-natured man in the prime of his life. Having ensured that all burdensome administrative tasks were taken care of by sedulous personnel, Timothy was free to spend his days to his own liking, namely to pursue his passion for learning and to satisfy his everlasting hunger for knowledge. He was not only the most well-read and universally erudite person far and wide at that time, he also had the heartfelt ambition to spread the spark of education among his fellow human beings, in particular the poorer folks that were subject of his.
```
Comment (adrigispen) [inline content/posts/2020-07-14-ancient-http/index.md:19]: in particular to the poorer folks who were subjects of his. 
in particular to his poorer subjects.
in particular to the poorer folk that were subject to him.
?

File: content/posts/2020-07-14-ancient-http/index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+As expected for the British establishment, Lord Timothy relished the comfort of a grave armchair and valued having a fresh cup of tea within arm’s reach above everything else. A lifestyle that by far not everyone was able to enjoy – least of all Timothy’s butler Isaac. While the Lord virtually never left his study room throughout the day, his butler constantly had to rush around between the endless bookshelves of the mansion’s private library. He was climbing up and down the creaking ladders, hastily gathering the books that the Lord wanted to browse through next. But as soon as Isaac had hauled a heavy pile of tomes over to the study room, Lord Timothy would send him back again with yet another list of titles to fetch.
```
Comment (adrigispen) [inline content/posts/2020-07-14-ancient-http/index.md:23]: Such a lifestyle was not available for all to enjoy - least of all Timothy's butler Isaac.

File: content/posts/2020-07-14-ancient-http/index.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+That, day after day, was how the years went by at the Learnersbee villa – until early October 1659. It was a drizzly, murky and overall dull afternoon (like most afternoons in Britain) and Lord Timothy was just about to finish a cup of mediocre second flush Darjeeling tea. His butler Isaac had already left for the library an hour ago to search for a couple of Wicitannica volumes. Lord Timothy, who felt a bit uneasy that day anyway (not to mention the rather underwhelming beverage), eventually lost patience. He heaved himself up from his armchair and shuffled over to the library to see “what the bloody Deuce” was going on. Upon his arrival he found that Isaac had ensconced himself on a carpet with his back leaning against the column of a bookshelf, blissfully snoring.
```
Comment (adrigispen) [inline content/posts/2020-07-14-ancient-http/index.md:25]: Upon his arrival he found Isaac ensconced on a carpet with his back against the column of a bookshelf, snoring blissfully.

Could leave it as is but the 'blissfully snoring' doesn't have a subject - reads a little strangely.

File: content/posts/2020-07-14-ancient-http/index.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+Lord Timothy wasn’t amused. Despite his exceptionally calm temperament and polite manners (even by British standards), he lost his composure for the fraction of a second. A twinge of sheer anger made him stamp his foot on the floor with such vigor that doors and windows were trembling. Before Isaac had a chance to realise what was going on, a small wooden box from one of the upper shelves came loose and fell down, right onto the butler’s head. Its lid broke open and small slips of paper by the hundreds were scattering all over the ground. The then following occurrence was precisely recorded in Isaac’s journal: Lord Timothy appeared to be glued to the spot with his eyes staring stonily at the floor. Then, after a long-lasting and awkward moment, he suddenly screamed “Eureka!” – and burst away, back to his study room, in which he locked himself in for 3 days in a row.
```
Comment (adrigispen) [inline content/posts/2020-07-14-ancient-http/index.md:27]: A twinge of sheer anger made him stamp his foot on the floor with such vigor that doors and windows trembled.

Its lid broke open and small slips of paper were scattered by the hundreds all over the ground.

and burst away to his study room, where he locked himself in for 3 days straight.

File: content/posts/2020-07-14-ancient-http/index.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+His invention got implemented county-wide within just a couple of months and can be best described as a postcard-based message exchange system. Centrepieces – and also widely recognisable symbols – were small, square-shaped kiosks, painted in glaring red, which got erected in the centre of every village. These kiosks had windows to all sides and were manned with a dispatcher during the daytime. Every citizen could just walk up to them in order to have a message recorded and sent out.[^2] The message itself was captured on the foreside of a small, roughly hand-sized piece of paper, along with some meta-information about sender and recipient. In order to save precious space on the card the notation followed a strictly standardised protocol, in which the different parts of the message were separated by hyphens (hence the term “hyphen text postcard”).
```
Comment (adrigispen) [inline content/posts/2020-07-14-ancient-http/index.md:33]: His invention was implemented county-wide

which were erected in the centre of every village.

File: content/posts/2020-07-14-ancient-http/index.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+Whoever was about to leave town – be it by foot, on a horseback or in a carriage, from the poorest beggar to the Lord himself – was officially obligated to stop by the local kiosk beforehand and pick up a batch of postcards. The clue was that these were not sorted by their final destination but rather by the *direction* they were supposed take. So if someone left town on an eastbound path, they would carry along all postcards that were designated to go somewhere in the East. Once the next village on the route was reached, the traveller would drop off the postcards at the respective kiosk, where they were looked through afresh and dispatched accordingly.
```
Comment (adrigispen) [inline content/posts/2020-07-14-ancient-http/index.md:35]: be it by foot, on horseback, or in a carriage,

File: content/posts/2020-07-14-ancient-http/index.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+It was a matter of honour for Lord Timothy to be registrant from the very start. He had advertised his private library, which, to his own surprise, people made excessive use of. Questions from all over the county arrived at the Learnersbee villa every day: curious kids wanted to know what there is behind the horizon, eager teachers asked about the latest developments in the Commonwealth, and self-taught healers wanted to learn about the effects of local herbs. Lord Timothy soon had to detail a savant whose sole responsibility was to research and answer the daily increasing volume of requests.
```
Comment (adrigispen) [inline content/posts/2020-07-14-ancient-http/index.md:41]: It was a matter of honour for Lord Timothy to be register himself from the very start.

Comment (adrigispen) [reply content/posts/2020-07-14-ancient-http/index.md:41]: He advertised his private library,

File: content/posts/2020-07-14-ancient-http/index.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+## A political wind of change
+
+The idea behind the hyphen text postcards was well ahead of its time and had the potential to be a societal game changer. Unfortunately, it didn’t even get a real chance to take off – it rather ended as aprupt as it had begun: Lord Timothy Learnersbee got dismissed in the light of the restoration of the Stuart monarchy in late 1660. Appointed successor and leader of the county thereupon was Lord Boris, Son of John II., an uncompromising traditionalist and ardent proponent of the monarchy, who didn’t find too much pleasure in the notion of an interconnected and independently thinking populace. The hyphen text postcards were doomed to extinction shortly after their rise, Lord Timothy’s faith is unbeknownst.
```
Comment (adrigispen) [inline content/posts/2020-07-14-ancient-http/index.md:45]: Lord Timothy Learnersbee was dismissed in the light of the..

Comment (adrigispen) [reply content/posts/2020-07-14-ancient-http/index.md:45]: The hyphen text postcards were doomed to extinction shortly after their rise, Lord Timothy’s faith is unbeknownst. << I don't totally know what you're trying to say here. this?

Thus the hyphen text postcards were doomed to extinction shortly after their rise. Lord Timothy’s fate is unknown.

File: content/posts/2020-07-14-ancient-http/index.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+
+All in all, this brief chapter in the British history is not just interesting from a purely technical point of view. It is also an impressive example that shows how infrastructure and technology in its broadest sense can have a powerful impact on society for the good of all. At the same time, however, the course of the subsequent events drastically proves how the success of forward-looking political ideas is largely dependent on the whims of those who rule. (And that, for a change, not just in Britain.)
```
Comment (adrigispen) [inline content/posts/2020-07-14-ancient-http/index.md:47]: however, the course of the subsequent events exemplifies how the success of forward-looking political ideas is largely dependent on the whims of those who rule.

File: content/posts/2020-07-14-ancient-http/index.md [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+
+
+[^1]: Just for the records: this entire story is made-up.
+[^2]: Most people were illiterate at that time, so it was the dispatcher’s job to put the text on paper.
+[^3]: Analogous to the writing, the dispatcher mostly had to read out the messages aloud upon delivery.
```
Comment (adrigispen) [inline content/posts/2020-07-14-ancient-http/index.md:52]: the dispatcher mostly had to read the messages aloud upon delivery.

File: content/posts/2020-07-14-ancient-http/index.md [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+[^4]: Changes and additions to the directory were broadcasted to the other kiosks regularly, also via postcards.
```
Comment (adrigispen) [inline content/posts/2020-07-14-ancient-http/index.md:53]: were broadcast

**Comment by jotaen** [comment]
thanks heaps @adrigispen ! this “get” instead of “is” really is an annoying habit, don’t know why I always do that 😆

**Pull request merged**